pkg/karmadactl/cmdinit/cmdinit.go

```diff
@@ -93,12 +93,8 @@ func NewCmdInit(parentCommand string) *cobra.Command {
 		},
 	}
 	flags := cmd.Flags()
-
-	releaseVer, err := version.ParseGitVersion(version.Get().GitVersion)
-	if err != nil {
-		klog.Infof("No default release version found. build version: %s", version.Get().String())
-		releaseVer = &version.ReleaseVersion{} // initialize to avoid panic
-	}
+	// image registry
+	flags.StringVarP(&opts.ImageRegistry, "private-image-registry", "", "", "Private image registry. Choose a container registry to pull control plane images from. Including all images required to karmada init, recommended for use when deploying karmada in privatized environments. This is a minimum priority parameter. For example, you can use kube-image-registry=registry.cn-hangzh to change the image related to kube")
```

Comment:
```suggestion
	flags.StringVarP(&opts.ImageRegistry, "private-image-registry", "", "", "Private image registry where pull images from. If set, all required images will be downloaded from it, it would be useful in offline installation scenarios.  In addition, you still can use --kube-image-registry to specify the registry for Kubernetes's images.")
```
How about this?

By the way, the comment(`// image registry`) above it seems redundant. The usage is clear enough, it can't give more details.

Reply:
Agree